kvbc/src/categorization/kv_blockchain.cpp

```diff
@@ -91,7 +91,6 @@ BlockId KeyValueBlockchain::addBlock(CategoryInput&& category_updates,
   last_raw_block.data.updates = category_updates;
   // Per category updates
   for (auto&& [category_id, update] : category_updates.kv) {
-    // https://stackoverflow.com/questions/46114214/lambda-implicit-capture-fails-with-variable-declared-from-structured-binding
     std::visit(
         [&new_block, category_id = category_id, &write_batch, &last_raw_block, this](auto& update) {
```

Comment:
I know this isn't part of this PR, but I just noticed it. Shouldn't the `update` parameter be taken by universal reference (`&&`) to signal a move. 

Similarly, the `update` argument should be wrapped in `std::move`, since it's an l-value reference.

Reply:
done